tools/config-gen/main.go

```diff
@@ -104,7 +108,7 @@ func main() {
 		FlagTemplateData: fd,
 		TypeTemplateData: td,
 		Backticks:        "`",
-	})
+	}, path.Join(*outDir, "config.go"), path.Join(*templateDir, "config.tpl"))
```

Comment:
nit: the formatting seems a bit odd here. These two (or at least one of these two) should have gone to the next row I think. Could you check?